test/unit_tests/datautil/test_windowers.py

```diff
@@ -126,6 +126,29 @@ def test_overlapping_trial_offsets(concat_ds_targets):
             drop_samples=False)
 
 
+def test_windows_from_events_preload_false(concat_ds_targets):
```

Comment:
If you want to have this test, then you could create dummy data and store in temp-folder. We should not have unfinished test in master I think

Reply:
Updated with newest commit!